ui/app/AppLayouts/Profile/views/MessagingView.qml

```diff
@@ -136,44 +136,52 @@ SettingsContentBase {
             components: [
                 StatusSwitch {
                     id: showMessageLinksSwitch
-                    checked: false
-                    onCheckedChanged: {
+                    function switchOffPreviewableSites() {
```

Comment:
I'd prefer having this (common) function outside of the delegate

Reply:
Good point. Will do

Reply:
Done.